Mathlib/Algebra/Algebra/Bilinear.lean

```diff
@@ -198,6 +198,13 @@ theorem _root_.Algebra.coe_lmul_eq_mul : ⇑(Algebra.lmul R A) = mul R A :=
 theorem _root_.Algebra.lmul_injective : Function.Injective (Algebra.lmul R A) :=
   fun a₁ a₂ h ↦ by simpa using FunLike.congr_fun h 1
 
+theorem _root_.Algebra.lmul_isUnit_iff {A : Type*} [CommSemiring A] [Algebra R A] (x : A) :
+    IsUnit (Algebra.lmul R A x) ↔ IsUnit x := by
+  refine ⟨fun h ↦ ?_, fun h ↦ h.map (Algebra.lmul R A)⟩
+  obtain ⟨x, hx⟩ := ((Module.End_isUnit_iff _).mp h).2 1
+  rw [Algebra.coe_lmul_eq_mul, LinearMap.mul_apply'] at hx
+  exact isUnit_of_mul_eq_one _ _ hx
```

Comment:
This is true for noncommutative A as well; if `lmul` is surjective there exists y such that xy=1, and if it's injective then xyx=1x=x=x1 implies yx=1 as well. We could add `IsUnit x ↔ Bijective (x * ·)` next to the def of IsUnit since it holds for any monoid.

Reply:
I added 
```lean
@[to_additive]
theorem isUnit_iff_mulLeft_bijective [Monoid M] {a : M} :
    IsUnit a ↔ Function.Bijective (a * ·) :=
```
and also a `mulRight` version and used that to prove `Algebra.lmul_isUnit_iff`. As you suggested, I added those lemmas in `Algebra.Group.Units` where `IsUnit` is defined even though one direction is proved in `Algebra.Group.Units.Equiv`: [Units.mulLeft_bijective](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Group/Units/Equiv.html#Units.mulLeft_bijective). I thought that since the proof was very short and it made more sense for the result to be next to the definition of `IsUnit` that was the better choice.